.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
+  - "iojs"
```

Comment:
Also, I believe io.js just released v2.0.0 yesterday. We may want to test against io.js v1 and v2 separately.
See https://github.com/mozilla/fxa-auth-db-server/pull/139/files